front/lib/resources/data_source_view_resource.ts

```diff
@@ -106,6 +106,17 @@ export class DataSourceViewResource extends ResourceWithVault<DataSourceViewMode
     });
   }
 
+  static async listForDataSource(
```

Comment:
Where is this function used?

Reply:
Ah sorry that was from another branch 😬 It slipped through, removing